packages/expo-camera/ios/CameraPermissionsRequester.swift

```diff
@@ -0,0 +1,136 @@
+import ExpoModulesCore
+import AVFoundation
+
+class CameraCameraPermissionRequester: NSObject, EXPermissionsRequester {
```

Comment:
It seems a bit redundant to have three classes that basically have a very subtle difference. Can we use inheritance here to remove the repeated code? Also a suggestion to name it `CameraOnlyPermissionRequester` or something like that instead of `CameraCameraPermissionRequester` 😅

Reply:
😅 That's definitely some copy and paste gone wrong